### DISCUSSION TITLE
Edit QSO

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#38] [Ideas]
To edit fields of a QSO I can select the field in the QSO table and edit it with 'Edit Value'.
In case of multiple changes, the field must be repeated for field.

Maybe 'Edit Value' could be changed to 'Edit QSO' and all QSO data of the record could be transferred to the lower part of the window for editing.

![grafik](https://user-images.githubusercontent.com/26311402/162791013-4637f2c3-e28f-4753-88ef-f168305bd619.png)

With "Save" the complete data record would than be taken over.

**Comment by foldynl** [comment]
I've thought about it a few times, but  I will stay with the current implementation. My goal was to separate edit and insert. 

But my opinion may change.
